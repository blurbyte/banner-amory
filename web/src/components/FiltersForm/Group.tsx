// simple wrapper to group filters

import styled from 'styled-components';

const Group = styled.div`
  display: flex;
  flex-wrap: wrap;

  &:not(:first-child) {
    margin-top: 1.2rem;
  }
`;

export default Group;
