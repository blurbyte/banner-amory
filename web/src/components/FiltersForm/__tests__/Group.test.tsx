import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Group from '../Group';

test('renders correctly', () => {
  const component = renderer.create(<Group />);
  expect(component.toJSON()).toMatchSnapshot();
});
