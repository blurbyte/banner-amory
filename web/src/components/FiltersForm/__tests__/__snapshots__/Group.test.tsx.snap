// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c0:not(:first-child) {
  margin-top: 1.2rem;
}

<div
  className="c0"
/>
`;
